app/helpers.php

```diff
@@ -1690,6 +1691,15 @@ function seeded_shuffle(array &$items, int $seed = 0)
     mt_srand();
 }
 
+function set_opengraph(HasOpengraph $model, ...$options)
+{
+    $className = 'App\\Libraries\\Opengraph\\'.get_class_basename(get_class($model)).'Opengraph';
+
+    View::share([
+        'opengraph' => (new $className($model, ...$options))->get(),
+    ]);
```

Comment:
this needs to be reset after each request because the value persists to the next requests until overwritten